src/archiefvernietigingscomponent/destruction/tests/test_tasks.py

```diff
@@ -213,6 +218,52 @@ def test_complete_and_notify(self, m):
         self.assertEqual(notification.user, destruction_list.author)
         self.assertEqual(notification.message, _("Processing of the list is complete."))
 
+    def test_complete_and_notify_process_owner(self, m):
+        process_owner = UserFactory.create(
+            role__type=RoleTypeChoices.process_owner,
+            role__can_review_destruction=True,
+            role__can_view_case_details=True,
+        )
+        destruction_list = DestructionListFactory.create(name="Summer List",)
+        DestructionListReviewFactory.create(
+            author=process_owner,
+            status=ReviewStatus.approved,
+            destruction_list=destruction_list,
+        )
+
+        destruction_list.process()
+        destruction_list.save()
+
+        complete_and_notify(destruction_list.id)
+
+        notifications = Notification.objects.all()
+
+        self.assertEqual(2, notifications.count())
+
+        notification = notifications.get(user=process_owner)
+
+        self.assertEqual(notification.destruction_list, destruction_list)
+
+        report = DestructionReport.objects.get()
+
+        self.assertEqual(
+            notification.message,
+            _(
+                "Destruction list Summer List has been processed. "
+                "You can download the report of destruction here: %(protocol)s://%(domain)s/reports/download/%(id)s/"
```

Comment:
```suggestion
                "You can download the report of destruction here: http://testserver/%(url)s"
```

`url = reverse("report:download-report")`